packages/image-utils/src/Image.ts

```diff
@@ -79,7 +79,8 @@ async function resizeAsync(imageOptions: ImageOptions): Promise<Buffer> {
           backgroundColor && backgroundColor !== 'transparent' ? backgroundColor : 'none'
         }" /></svg>`
       );
-      sharpBuffer.composite([{ input: mask, blend: 'dest-over' }]);
+
+      sharpBuffer.composite([{ input: mask, blend: 'dest-in' }]);
```

Comment:
what's the purpose of this change?

Reply:
This is the more dangerous change, it causes. the circle to actually be applied, without this the image is still square. But the change isn't perfect, in some cases I observed transparent images being created.